packages/react-components/src/utils/text.ts

```diff
@@ -1,4 +1,9 @@
-export const appendSuffix = (val: string, suffix: string): string =>
+export const getCounterString = (count: number, term: string): string => {
+  const int = Number(count.toFixed());
```

Comment:
well technically if we could do 1.5 Lab it would still be singular. I notice that you are not testing decimals yet using `toFixed()` - I guess since there is no way to enforce an `Integer` in TypeScript through types so we have to either throw or round like you did. I would suggest to test the decimals